src/app/mi-details/view-story/view-story.page.html

```diff
@@ -0,0 +1,127 @@
+<app-application-header [config]="headerConfig">
+    <ion-buttons class="button-container" header-right-content>
+      <ion-button class="button" (click)="share()">
+        <ion-icon name="share-social-outline" class="icon"></ion-icon>
+        <span class="text">{{"SHARE" | translate}}</span>
+      </ion-button>
+      <ion-button class="button" (click)="download()">
+        <ion-icon name="download-outline" class="icon"></ion-icon>
+        <span class="text">{{"DOWNLOAD"| translate}}</span>
+      </ion-button>
+    </ion-buttons>
+</app-application-header>
+
+<ion-content class="ion-padding" *ngIf="viewProjectDetails">
+  <div class="layout-wrapper">
+
+  <div class="author-container ion-padding">
+    @if(storyDetails.title){
+      <h1 class="title-header">{{storyDetails.title}}</h1>
+    }
+    <div class="author-text">
+      @if(storyDetails.authorName){
+      <span>{{"BY" | translate}} </span>
+      <div class="author-span">
+        <span>{{storyDetails.authorName}}</span>
+        <span>{{storyDetails.location}}</span>
+      </div>
+      }
+    </div>
+  </div>
+
+@if(storyDetails.summary){
+  <div class="summary-container">
+    <div class="summary-head">
+      <h1 class="summary-title">{{"SUMMARY" | translate}}</h1>
+      <button class="summary-copy" (click)="copyText()">
+        <ion-icon name="copy-outline"></ion-icon>
+      </button>
+    </div>
+    <p class="summary-text">{{storyDetails.summary}}</p>
+  </div>
+}
+  
+  @if(storyDetails.problemStatement){
+    <app-mi-details-card
+        [cardConfig]="{ title: 'PROBLEM_STATEMENT', text: storyDetails.problemStatement }">
+    </app-mi-details-card>
+  }
+
+
+@if(storyDetails.objective){
+  <app-mi-details-card 
+    [cardConfig]="{ title: 'OBJECTIVE', text: storyDetails.objective }">
+  </app-mi-details-card>
+}
+
+@if(storyDetails.timeline){
+  <app-mi-details-card 
+    [cardConfig]="{ title: 'TIMELINE', text: storyDetails.timeline}">
+  </app-mi-details-card>
+}
+
+@if(viewProjectDetails.tasks.length>0){
+  <app-mi-details-card
+    [cardConfig]="{ title: 'ACTION_STEPS', tasks: viewProjectDetails.tasks }">
+  </app-mi-details-card>
+}
+
+@if(storyDetails.impact){
+  <app-mi-details-card 
+    [cardConfig]="{ title: 'IMPACT', text: storyDetails.impact }">
+  </app-mi-details-card>
+}
+
+@if(viewProjectDetails.attachments.length > 0){
+  <div class="resource-container">
+    <h1 class="resource-title">
+      <ion-icon name="star"></ion-icon>
+      {{"RESOURCES" | translate}}
+    </h1>
+    @if(resource.images.length > 0){
+      <div class="resource-item">
+        @for(image of resource.images;track image){
+          <div class="item">
+            <ion-icon name="image-outline"></ion-icon>
+            <a [href]="image.url" download="{{image.url}}">
+              <span>{{image.name}}</span>
+            </a>
+          </div>
+        }
+      </div>
+    }
+    @if(resource.documents.length > 0){
```

Comment:
use conditions like resource.documents.length instead of resource.documents.length > 0 wherever similar conditions are and use safety operator to access the properties from an object